include/query.php

```diff
@@ -28,7 +28,7 @@
 // Setup input variables..
 $boolean_search = isset($_GET['boolean']) ? (int) $_GET['boolean'] : 0;
 $status = !empty($_GET['status']) ? (string)$_GET['status'] : 'Open';
-$search_for = !empty($_GET['search_for']) ? (string)$_GET['search_for'] : '';
+$search_for = !empty($_GET['search_for']) ? (string)trim($_GET['search_for']) : '';
```

Comment:
That code should be revised anyway, but to keep the intended meaning, and to avoid an error under PHP 8, the string cast should be applied to the $_GET variable:
```suggestion
$search_for = !empty($_GET['search_for']) ? trim((string)$_GET['search_for']) : '';
```